docs/changelog.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
```

Comment:
Should we do this the other way round instead, i.e. latest date at the top (like we do for release notes)? I'm not as familiar with change logs in general but it would make sense to me to switch the order.

Reply:
That's a great idea. Reversed the order.